sql/mob_droplist.sql

```diff
@@ -16603,6 +16603,15 @@ INSERT INTO `mob_droplist` VALUES (3146,0,0,1000,15123,10); -- Valor Breeches
 INSERT INTO `mob_droplist` VALUES (3146,0,0,1000,15140,10); -- Monster Gaiters
 INSERT INTO `mob_droplist` VALUES (3146,0,0,1000,15142,10); -- Scout's Socks
 INSERT INTO `mob_droplist` VALUES (3146,0,0,1000,16360,10); -- Etoile Tights
+INSERT INTO `mob_droplist` VALUES (3147,0,0,1000,5564,1000); -- Hydra ; Hydra meat  210%
```

Comment:
Purely a style thing, but using ';' as a delimiter here makes me a little uneasy because it's a valid character for SQL. Can you change this to one of the other styles in the file?
`(Hydra) Hydra Meat  210%`
`Hydra -- Hydra Meat  210%`

Reply:
K done